spec/calendar.html

```diff
@@ -99,9 +99,8 @@ <h1>CalendarFields ( _calendar_, _fieldNames_ )</h1>
       </p>
       <emu-alg>
         1. Let _fields_ be ? GetMethod(_calendar_, *"fields"*).
-        1. Let _fieldsArray_ be CreateArrayFromList(_fieldNames_).
-        1. If _fields_ is not *undefined*, then
-          1. Set _fieldsArray_ to ? Call(_fields_, _calendar_, « _fieldsArray_ »).
+        1. If _fields_ is *undefined*, return _fieldNames_.
+        1. Let _fieldsArray_ be ? Call(_fields_, _calendar_, « CreateArrayFromList(_fieldNames_) »).
         1. Return ? IterableToListOfType(_fieldsArray_, « String »).
```

Comment:
I think the current spec observably gets `Array.prototype[@@iterator]`. Is that not the case?